libs/shared/ui/src/lib/components/scroll-shadow-wrapper/scroll-shadow-wrapper.tsx

```diff
@@ -49,7 +49,7 @@ export function ScrollShadowWrapper(props: ScrollShadowWrapperProps) {
       data-testid="scroll-shadow-wrapper"
       ref={wrapperRef}
       style={style}
-      className={`relative overflow-auto ${className}`}
+      className={`relative overflow-y-auto pr-[1px] ${className}`}
```

Comment:
I added this small pr-[1px] because the overflow-y-auto also overflow-x and this overflow-x truncate the outline when you select an element which ends with a pretty ugly rendering.
I would love to add overflow-x-visible but as soon as you add overflow-auto on x or y, it adds it for both and you can't change that...